importers/src/main/groovy/whelk/importer/WhelkCopier.groovy

```diff
@@ -19,13 +23,37 @@ class WhelkCopier {
         this.dest = dest
         this.recordIds = recordIds
 
-        dest.storage.versioning = false
         dest.storage.doVerifyDocumentIdRetention = false
     }
 
     void run() {
         TreeSet<String> alreadyImportedIDs = new TreeSet<>()
 
+        // Import all (ish) auth records
+        for (relDoc in selectBySqlWhere("collection = 'auth' and data#>>'{@graph,1,@type}' in (\n" +
```

Comment:
Om vi parameteriserar denna manöver och skicka in typer som ska kopieras i sin helhet istället, så kan vi styra det per miljö och modifiera urvalet över tid utan att behöva koda om denna snurra. (Att ange collection blir då rimligt redundant och kan skippas. Vi *bör* förlita oss på indexet på `@type` (`idx_lddb_type`).)

Reply:
Det verkar aldrig ha funnits nåt index på @type :) Men processen fungerar hjälpligt ändå. Jag har pushat ändringarna du bad om.

Reply:
Stämmer, ser ut som om `librisxl-tools/postgresql/migrations/00000004-add-indexes.plsql` aldrig har applicerats?

Reply:
Jag misstänker att det är en lokal fil du har. I repot finns bara:
00000004-remove-unused-indices.plsql

Reply:
Haha, det stämmer! Läge att lägga på ett sådant då, det är ju en ganska central sak.